src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToTargetEolFrameworks.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Linq;
+using FluentAssertions;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Build.Tests
+{
+    public class GivenThatWeWantToTargetEolFrameworks : SdkTest
+    {
+        public GivenThatWeWantToTargetEolFrameworks(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        [Theory]
+        [InlineData("netcoreapp1.0")]
+        [InlineData("netcoreapp1.1")]
+        [InlineData("netcoreapp2.0")]
+        [InlineData("netcoreapp2.2")]
+        [InlineData("netcoreapp3.0")]
+        public void It_warns_that_framework_is_out_of_support(string targetFrameworks)
```

Comment:
These "start from msbuild /t:build" tests are slow. I think just sample one TFM is enough. @dsplaisted what do you think?

Reply:
Good to know. I mostly did this to make sure we cover all the EOL frameworks and didn't forget to add one, but if it impacts perf I'm fine scaling this back

Reply:
We have a ton of tests that work like this, I'm not sure if individually they're especially slow.  But I also think that one or two TFMs is enough for coverage here.